packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -156,21 +156,6 @@ export default class RunPanelStore {
     }
 
     async onRunButtonClick() {
-        let timer_counter = 1;
-        if (window.sendRequestsStatistic) {
-            performance.clearMeasures();
```

Comment:
@farabi-deriv we need the setInterval I guess

Reply:
Ok 👍